src/routes/api/payments/updateMember/+server.ts

```diff
@@ -123,9 +128,9 @@ ${MEMBER_API_URL}/organization/${MEMBER_ORGANIZATION}/members\
  * Generated partially with ChatGPT, so examine carefully
  */
 
-function standardizeAddress(address: string) {
+function standardizeAddress(address: string): { houseNumber: string; streetName: string; direction: string | undefined } {
   // Regular expression to match the house number and street name
-  const regex = /(\d+)\s*(?:NW|NE|SW|SE)?\s*(\d+)?(?:st|nd|rd|th)?\s*(\w+)/i;
+  const regex = /^(?<houseNumber>\d+)\s*(?<direction>NW|NE|SW|SE)?\s*(?<streetName>\w+)?\s*(?<streetType>\w*)/i;
```

Comment:
are there no streets that would have just the basic cardinal directions? `N|W|E|S` ?